PayoneerCheckout.podspec

```
@@ -8,14 +8,8 @@ Pod::Spec.new do |s|
   s.author           = { 'Payoneer Germany GmbH' => '' }
   s.source           = { :git => 'https://github.com/optile/checkout-ios.git', :tag => s.version.to_s }
 
-  s.ios.deployment_target = '12.4'
-  s.swift_version = '5.3'
+  s.ios.deployment_target = '12.0'
+  s.swift_version = '5.5'
 
   s.source_files = 'Sources/**/*.swift'
-  
-  s.resources = ['Sources/Assets/*']
-
-  # s.public_header_files = 'Pod/Classes/**/*.h'
-  s.frameworks = 'UIKit'
```

Why did you delete this line?

```suggestion
  s.frameworks = 'UIKit'
```

---

AFAIK this causes CocoaPods to include UIKit in the global header, meaning all files in the SDK implicitly import UIKit. Everything works fine after removing it, but there's very little documentation on this, so if you have more info let me know.

---

At least here (https://guides.cocoapods.org/syntax/podspec.html#frameworks) I see that we should include all system framework we use. Despite the fact that I believe that everything can work without it I think it's safer to include it.